template.hbs

```diff
@@ -345,245 +355,17 @@
-          <h3 class="third-level-title">
-            {{ i18n @key "otherFields" ../../Certificate.CertificateLanguages }}</h3>
-          {{#each this}}
-          {{!-- KeyValueObject --}}
-          <tr class="table-rows">
-            <th class="table-headers" colspan="2">{{@key}} {{this.Key}}</th>
-            <td class="table-cells">{{ localizeValue this.Value this.Type
-              ../../../Certificate.CertificateLanguages.[0]
-              }}
-              {{this.Unit}}
-            </td>
-            <td class="table-cells">{{this.Interpretation}}</td>
-          </tr>
-          {{/each}}
-        </tbody>
-      </table>
-      {{else}}
-
-      {{!-- XTest-> Measurement(s) ( C42) --}}
-      {{#ifEqual @key "C42"}}
-      <table class="main-table" style="margin-top: 0px">
-        <tbody>
-          <tr class="table-rows">
-            <th class="table-headers" colspan="2">{{@key}}
-              {{ i18n @key "certificateFields" ../../Certificate.CertificateLanguages }}
-            </th>
-            <td class="table-cells" colspan="2">
-              {{ joinAndLocalizeNumber this ", " ../../Certificate.CertificateLanguages.[0] "Value" }} {{
-              this.[0].Unit
-              }}
-            </td>
-          </tr>
-        </tbody>
-      </table>
-
-      {{!-- XTest-> Measurement(s) (C10, C11, C12, C13, C31, C32, C41, C43, C44) --}}
-      {{else}}
-      <table class="main-table" style="margin-top: 0px">
-        <tbody>
-          <tr class="table-rows">
-            <th class="table-headers" colspan="2">{{@key}}
-              {{ i18n @key "certificateFields" ../../Certificate.CertificateLanguages }}
-            </th>
-            <td class="table-cells">
-              {{ localizeNumber this.Value ../../Certificate.CertificateLanguages.[0] }} {{this.Unit}}
-            </td>
-            <td class="table-cells">
-              {{#if this.Minimum}}
-              min {{localizeNumber this.Minimum ../../Certificate.CertificateLanguages.[0]}} {{this.Unit}}
-              {{/if}}
-              {{#if this.Maximum}}
-              max {{localizeNumber this.Maximum ../../Certificate.CertificateLanguages.[0]}} {{this.Unit}}
-              {{/if}}
-            </td>
-          </tr>
-        </tbody>
-      </table>
-      {{/ifEqual}}
-      {{/ifEqual}}
-
-      {{!-- XTest-> C10, C30, C40 --}}
-      {{else}}
-      <table class="main-table" style="margin-top: 0px">
-        <tbody>
-          <tr class="table-rows">
-            <th class="table-headers" colspan="2">{{@key}}
-              {{ i18n @key "certificateFields" ../../Certificate.CertificateLanguages }}
-            </th>
-            <td class="table-cells" colspan="2">{{this}}</td>
-          </tr>
-        </tbody>
-      </table>
-      {{/typeOf}}
-      {{/each}}
-      {{/in}}
-    </section>
-
-    {{!-- Inspection-> C00, C01, C02, C03 --}}
+    {{> inspection Inspection=this CertificateLanguages=../Certificate.CertificateLanguages }}
```

Comment:
instead of copying the huge Inspection, it's now in a dedicated partial.
See how to use it in `generate-html-cli.js`.

Reply:
Awesome